CHANGELOG.md

```diff
@@ -1,5 +1,108 @@
 # Unity Converters for Newtonsoft.Json changelog
 
-## x.0.1
+## 1.0.0
 
 - ✨ Initial release
+
+- ❤ Big love from these authors allowing me to take inspiration from their
+  converters packages:
+
+  - ParentElement <https://github.com/ianmacgillivray/Json-NET-for-Unity>
+  - Wanzyee Studio <http://wanzyeestudio.blogspot.com/2017/03/jsonnet-converters.html>
+
+- UnityConverterInitializer that registers converters on load
+
+- Ensured via tests to work in Unity version:
+  - 2018.4
+  - 2019.2
+  - 2020.1
+
+- Ensured via tests to work with API compatibility:
+  - .NET Standard 2.0
+
+- Ensured via tests to work with Newtonsoft.Json version:
+  - 12.0.3
+
+- Added custom converters for following types:
+  - [x] (AI/NavMesh) UnityEngine.AI.NavMeshQueryFilter
+  - [x] (AI/NavMesh) UnityEngine.AI.NavMeshTriangulation
+  - [x] (Camera) CullingGroupEvent
+  - [x] (Geometry) UnityEngine.Bounds
+  - [x] (Geometry) UnityEngine.BoundsInt
+  - [x] (Geometry) UnityEngine.Plane
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461394)